doc/03_reference/coverage_plan.rst

```diff
@@ -352,3 +352,5 @@ There must be a documented reason a particular bin is added to the illegal or ig
 * ``pmp_iside_priv_bits_cross``, ``pmp_iside2_priv_bits_cross``, ``pmp_dside_priv_bits_cross``, PMP regions x permissions x access fail/pass x privilege level
 
   * Three crosses, one for each PMP channel (instruction, instruction 2 and data).
+
+* ``misaligned_insn_bus_err_cross``, ``misaligned_data_bus_err_cross`` - Implemented to check if we see errors from memory interfaces (two different crosses for data and instruction memory) in the case of a misaligned access.
```

Comment:
No need to add this here, consider the Cross Coverage section to be all those crosses that aren't already mentioned.